junk/gen-bench-data/gen_benchmark_data.sh

```diff
@@ -7,14 +7,20 @@ set -e
 echo "Pre-build a binary..." >&2
 cargo build --quiet --release
 
-for bench in multiple-cids10 \
-             multiple-peers8 \
-             multiple-sigs30 \
-             dashboard network-explore; do
+for bench in canon-map-key-by-lens \
+            canon-map-key-element-by-lens \
```

Comment:
1. I think it worth adding a similar bench for simple stream as a baseline.
2. I think it worth explicitly mentioning a problem/data size in the bench name (we may want to have same bench instances of different sizes).

Reply:
Agree but IMHO it should be done with a separate patch.

Reply:
Problem size should be added as early as possible.